docs/section19/19-2.md

```diff
@@ -0,0 +1,74 @@
+# Section 19 - Rule 19-2   
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-2     
+**Rule Description:** Baseline building plant capacities shall be based on coincident loads.    
+**Rule Assertion:** Options are Pass/Fail/IN_APPLICABLE     
+**Appendix G Section:** Section G3.1.2.2    
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each FluidLoop Data Group  
+
+**Applicability Checks:** 
+
+1. Applies to baseline buildings with chilled water loops served by chiller(s) to provide space cooling and/or buildings with HW served by boiler(s) to provide space heating.
+
+**Function Calls:**  
+1. is_hvac_sys_fluid_loop_attached_to_boiler()
+2. is_hvac_sys_cooling_type_fluid_loop()
+3. is_hvac_sys_heating_type_fluid_loop()
+4. is_hvac_sys_fluid_loop_attached_to_chiller()
+5. is_hvac_sys_preheating_type_fluid_loop()  
+6. is_hvac_sys_preheat_fluid_loop_attached_to_boiler()  
+7. are_all_terminal_heating_loops_attached_to_boiler()
+
+
+
+## Rule Logic:   
+- Cycle through each HVAC system in the B-RMR to create a list of HW loops for space heating served by boiler(s) and a list of CHW loop for space cooling served by chiller(s) and a list of condenser water loops: `for each hvac in B_RMR...HeatingVentilatingAirConditioningSystem:`  
+    - Check if the hvac system heating coil is a fluid loop: `if is_hvac_sys_heating_type_fluid_loop(B_RMR, hvac.id)  == TRUE:`  
+        - Check if the fluid loop is attached to a boiler, if yes then add to list of HW fluid loops: `if is_hvac_sys_fluid_loop_attached_to_boiler(B_RMR, hvac.id) ==  TRUE: HW_fluid_loop_list =  HW_fluid_loop_list.append(hvac.heating_system.hot_water_loop)` 
```

Comment:
if using append, we do not need to add `HW_fluid_loop_list = `. This line can be modified:
`if is_hvac_sys_fluid_loop_attached_to_boiler(B_RMR, hvac.id) == TRUE: HW_fluid_loop_list.append(hvac.heating_system.hot_water_loop)`

Reply:
Thanks!